service/src/main/java/bio/terra/workspace/service/resource/controlled/model/ControlledResource.java

```diff
@@ -305,6 +306,8 @@ public ControlledResourceFields buildControlledCloneResourceCommonFields(
     // override name and description if provided
     cloneResourceCommonFields.name(name == null ? getName() : name);
     cloneResourceCommonFields.description(description == null ? getDescription() : description);
+    cloneResourceCommonFields.applicationId(
+        applicationId == null ? getApplicationId() : applicationId);
```

Comment:
What does `getApplicationId` do in this case? I'm trying to understand why there would be an application ID if one was not passed in.

Reply:
Yeah this isn't needed as a parameter; all of the callsites that thread it through are grabbing it from the source anyway. I've removed the param.